14/umbraco-cms/customizing/section-trees/sections/section-view.md

```diff
@@ -12,17 +12,103 @@ This page is a work in progress and may undergo further revisions, updates, or a
 
 <figure><img src="../../../.gitbook/assets/section-views.svg" alt=""><figcaption><p>Section View</p></figcaption></figure>
 
-**Manifest**
+## Creating a custom Section View
```

Comment:
Let's give this guide some context!

```suggestion
## Creating a custom Section View

In this section, you can learn how to register and create a custom Section View for the Umbraco backoffice.
```

Reply:
Oh, i didnt know there was a "commit suggestion" button  😲